ldapauth.go

```diff
@@ -112,6 +118,22 @@ func New(ctx context.Context, next http.Handler, config *Config, name string) (h
 
 	LoggerINFO.Printf("Starting %s Middleware...", name)
 
+	if config.BindDN != "" && config.BindPassword == "" {
+		bindPasswordLabel := "LDAP_AUTH_BIND_PASSWORD"
```

Comment:
Is there a reason why lines 122-126 could not be simplified to:
`config.BindPassword = getSecret(config.BindPasswordLabel)` ?

config.BindPasswordLabel is already set to default value `LDAP_AUTH_BIND_PASSWORD`, so if it ends up being empty, it is that the user purposely did that.